### PR TITLE
doc: suggest using timeoutstopsec in systemd file during IBD

### PR DESCRIPTION
**Pull request by ghost**
A naive user may run into trouble when using the provided systemd service file. By default, systemd will kill a process if it hasn't stopped after 90 seconds, which may be to short during Initial Block Download. This PR adds a suggestion to increase the timeout during IBD.

### CODE REVIEW

**Comment by d3spwn** [comment]
> You mean that during IBD it doesn't quit with SIGINT within 90sec?

Yes, it may not. Especially with slow hardware and/or large dbcache (which is often recommended for IBD).

File: doc/init.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ To test, run `systemctl start bitcoind` and to enable for system startup run
 
 NOTE: When installing for systemd in Debian/Ubuntu the .service file needs to be copied to the /lib/systemd/system directory instead.
 
+NOTE: Systemd can be very agressive in killing processes that don't stop in a reasonable amount of time. This can be a problem during Initial Block Download where it can take several minutes to flush cached data to disk. Therefore consider adding `TimeoutStopSec=infinity` to the file during IBD.
```
Comment (practicalswift) [inline doc/init.md]: I don't mean to be aggressive but you're missing a "g" in "aggressive"111

Comment (d3spwn) [reply doc/init.md]: Sorry. Fixed.

**Comment by practicalswift** [comment]
Perhaps the default value should be changed?

What would be a reasonable default for `TimeoutStopSec` that can be expected to work also during IBD in the general case?

**Comment by GChuf** [comment]
Maybe the default timestopsec value should be changed in another PR?
This could get merged for now.

**Pull request closed without merging**